ryven-editor/ryven/main/packages/nodes_package.py

```diff
@@ -13,7 +13,7 @@
 from ryvencore import Node, Data
 
 from ryven.main.utils import read_project, ryven_dir_path, abs_path_from_package_dir
-
+from .gui_env import in_gui_mode, load_current_guis
```

Comment:
same here, you should not import anything from `ryven.gui` in `ryven.main` unless you know you're running in GUI mode